changes.d/6623.fix.md

```diff
@@ -0,0 +1,3 @@
+Auto restart: The option to condemn a host in "force" mode (that tells
+workflows running on a server to shutdown as opposed to migrate) hasn't worked
```

Comment:
```suggestion
Auto restart: The option to to tell 
workflows running on a server to shutdown as opposed to migrate hasn't worked
```

Reply:
This PR will continue to use "force mode" as documented:

https://cylc.github.io/cylc-doc/stable/html/plugins/main-loop/built-in/cylc.flow.main_loop.auto_restart.html#module-cylc.flow.main_loop.auto_restart